src/HomeWork3/FirstTrainingManualToEnd/FirstTrainingManualToEnd.java

```diff
@@ -0,0 +1,355 @@
+        } else if (x > 1 && x <= 4) {
+            System.out.println(x + " рубля");
+        } else if (x > 4 && x < 28) {
+            System.out.println(x + " рублей");
+        }
+        System.out.println("");
+        /*
+        * Task #9
+        *
+            Задание 9
+            Изменить пример с суммой чисел таким образом, чтобы рассчитывалась не сумма, а
+            произведение, т.е. факториал числа.
+        * */
+        System.out.println("Task #9");
+//?????????????????????????????????????????????????????????????????????????????????????????????????????????????????????
+        double sumNumbers = 1;
+        int randomNumbers = (int) (Math.random() * 20);
+        int count = 0;
+
+        while (randomNumbers != 0) {
+            sumNumbers *= randomNumbers;
+            count++;
+            randomNumbers = (int) (Math.random() * 20);
+            if (count != 0) {
+                System.out.println(count + ": " + sumNumbers);
+            } else {
+                System.out.println("Косяк на касяке");
+            }
+        }
+
+//
+        System.out.println("");
+        /*
+        * Task # 10
+        *Задание 10
+         Посчитать факториал числа в границах от 10 до 15 (не используя рекурсию).
+      *
+        * */
+        System.out.println("Task #10");
+        int resultFactor = 1;
+        for (int i = 10; i <= 15; i++) {
+            resultFactor *= i;
+        }
+        System.out.println("Факториал :" + resultFactor);
+        System.out.println("");
+        /*
+        * Task #11
+        *Имеется целое число, определить является ли это число простым, т.е.
+        делится без остатка только на 1 и себя.
+        * */
+        System.out.println("Task #11");
+
+        int naturalNumber = (int) (Math.random() * 45);
+
+        if ((naturalNumber % naturalNumber) == 0 && (naturalNumber % 1) == 0) {
+            System.out.println("Число " + naturalNumber + " простое");
+        } else {
+            System.out.println("Число не простое");
+        }
+        System.out.println("");
+        /*
+         * Task #12
+         * Найдите сумму первых n целых чисел, которые делятся на 3.
+         * */
+        System.out.println("Task #12");
+        int integerNumbers = (int) (Math.random() * 32);
+        int sumIntegerNumbers = 0;
+        for (int i = 1; i < 10; i++) {
+            if ((integerNumbers % 3) == 0) {
+                sumIntegerNumbers += integerNumbers;
+
+            }
+
+        }
+        System.out.println("сумму первых n целых чисел, которые делятся на 3 = " + sumIntegerNumbers);
+        System.out.println("");
+        /*
+        * Task #13
+        * Создать последовательность случайных чисел, найти и вывести наибольшее
+        из них.
+        * */
+        System.out.println("Task #13");
+        int[] randomNumber = new int[16];
+        int maxValueNumber = 0;
+
+        for (int index = 0; index < randomNumber.length - 1; index++) {
```

Comment:
if we check < randomNumber.length - 1, we don't check last number, stop on pre-last